src/hotspot/share/prims/jvmtiRedefineClasses.cpp

```diff
@@ -932,6 +932,11 @@ jvmtiError VM_RedefineClasses::compare_and_normalize_class_versions(
       (the_class->super() == NULL || scratch_class->super() == NULL ||
        the_class->super()->name() !=
        scratch_class->super()->name())) {
+    log_trace(redefine, class, normalize)
+      ("redefined class %s superclass change error: superclass changed from %s to %s.",
```

Comment:
I'm not so sure I like the style of the log message wording. I would prefer something like:

     error redefining class %s: superclass changed from %s to %s.

The same comment applies to pretty much every error message below. Besides being more concise and easier to read, I don't like using "superclass change error" when there is no such formally defined error. The error is `JVMTI_ERROR_UNSUPPORTED_REDEFINITION_HIERARCHY_CHANGED`.

Reply:
I see now that you are just being consistent with the style used in existing logs, so I guess it's ok.